### PR TITLE
Remove iostream include

### PR DESCRIPTION
**Pull request by endJunction**
Replace cout/cerr output with hopefully semantically equal logog calls.

### CODE REVIEW

File: BaseLib/IO/Writer.cpp [hunk 1]
```diff
@@ -12,13 +12,13 @@
...
-// ** INCLUDES **
-#include "Writer.h"
-
-#include <iostream>
 #include <fstream>
 #include <limits>
 
+#include <logog/include/logog.hpp>
+
+#include "Writer.h"
```
Comment (norihiro-w) [inline BaseLib/IO/Writer.cpp]: why include its header here?

**Comment by chleh** [comment]
:+1:

File: BaseLib/IO/Writer.cpp (hunk 1 above)
Comment (endJunction) [inline BaseLib/IO/Writer.cpp]: It was a momentary weakness ;)

**Comment by TomFischer** [comment]
:fast_forward:

**Pull request merged**